Formula/code-server.rb

```diff
@@ -66,9 +66,9 @@ def plist
   end
 
   test do
-    # See https://github.com/cdr/code-server/blob/master/ci/build/test-standalone-release.sh
-    system bin/"code-server", "--extensions-dir=.", "--install-extension", "ms-python.python"
-    assert_match "ms-python.python",
+    # See https://github.com/cdr/code-server/blob/main/ci/build/test-standalone-release.sh
+    system bin/"code-server", "--extensions-dir=.", "--install-extension", "wesbos.theme-cobalt2"
+    assert_match "wesbos.theme-cobalt2",
       shell_output("#{bin/"code-server"} --extensions-dir=. --list-extensions")
```

Comment:
```suggestion
      shell_output("#{bin}/code-server --extensions-dir=. --list-extensions")
```
While we're looking at the test, it would be good to clean up this line a bit.